src/main/resources/Task 1 - Shoe Object Modelling.txt

```diff
@@ -0,0 +1,5 @@
+Object Modeling Shoes:
+- create a class diagram for shoes object hierarchy.
```

Comment:
on diagram if kidShoe extends Shoe than the arrow should point different direction